src/List/ItemList.js

```diff
@@ -22,22 +24,48 @@ const Items = props => {
     }
   }, [token, setdbItems, dbItems, props]);
 
-  //This will update an item to include a purchase date
-  const handleChange = e => {
-    let datePurchased = new Date();
-    let db = firebase.fb.firestore();
-    db.collection(token)
-      .doc(e.target.value)
-      .update({ datePurchased });
-    console.log(dbItems);
+  // Checks if a purchase date already exists - if not, creates a purchased date and increments # of purchases - if so, also sets the most recent purchase estimate, the most recent purchase interval and the calculated date of the next purchase
```

Comment:
Small note, move this into a multi-line comment instead of making the whole thing one line. Harder to read this way!